### PR TITLE
Biggest Attack on Internet History

### PR DESCRIPTION
**Pull request by unicodeveloper**

### CODE REVIEW

File: _posts/2016-10-25-how-the-biggest-attack-on-the-internet-was-perpertrated.markdown [hunk 1]
```diff
@@ -0,0 +1,66 @@
+---
+layout: post
+title: "How was the biggest attack on the Internet history perpetrated?"
```
Comment (jeffreylees) [inline _posts/2016-10-25-how-the-biggest-attack-on-the-internet-was-perpertrated.markdown]: For a title I'd reword to "How was the biggest attack in Internet history perpetrated?" or something that flows a little better.

File: _posts/2016-10-25-how-the-biggest-attack-on-the-internet-was-perpertrated.markdown [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+- dyn
+related:
+- 2016-07-11-pokemon-go-catches-all-your-data
+- 2015-12-28-securing-your-nodebots-with-json-web-tokens
+- 2015-11-06-surprised-turns-out-consumers-dont-trust-iot-security/
+---
+
+---
+
+**TL;DR:** Last week, one of the biggest attacks in the internet history was perpetrated via a group of hackers. Popular websites like **Twitter**, **Netflix**, **GitHub**, **Amazon**, **Spotify** were knocked offline. These web sites were not accessible for the period the attacks lasted. A Distributed Denial of Service (DDOS) attack on Dyn - a domain name server company affected the online availability of these popular platforms.
```
Comment (jeffreylees) [inline _posts/2016-10-25-how-the-biggest-attack-on-the-internet-was-perpertrated.markdown]: Might read a little better with "during which" or something, and added a hyphen to end the parenthetical in the second sentence.

"These web sites were not accessible for the period during which the attacks lasted. A Distributed Denial of Service (DDOS) attack on Dyn - a domain name server company - affected the online availability of these popular platforms."

File: _posts/2016-10-25-how-the-biggest-attack-on-the-internet-was-perpertrated.markdown [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+---
+
+## How did it happen?
+
+Investigations about the recent DDOS attacks primarily revolved around **Internet of Things(iOT)**. It was revealed that several cameras, digital video recorders(DVRs) were hacked to create a massive botnet that struck Dyn with lots of traffic, about a trillion bits of data per second. These cameras and devices were vulnerable because they used default and common guessable passwords.
```
Comment (jeffreylees) [inline _posts/2016-10-25-how-the-biggest-attack-on-the-internet-was-perpertrated.markdown]: Make sure and check usages of "Internet of Things" and the acronym "IoT" for consistent capitalization across the post.

File: _posts/2016-10-25-how-the-biggest-attack-on-the-internet-was-perpertrated.markdown [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+The attack on Dyn came from tens of millions of IP addresses according to the formal statement that was released on Dyn's website. The botnet that invoked the attack looks for smart home and **Internet of things** devices and transforms them into bots to use in cyberattacks. It was also discovered that the primary manufacturer of these devices utilized the default username and password combination such as `root` and `xc3511` respectively. These credentials are common on the Internet and can be used to access various types of DVRs.
```
Comment (jeffreylees) [inline _posts/2016-10-25-how-the-biggest-attack-on-the-internet-was-perpertrated.markdown]: > It was also discovered that the primary manufacturer of these devices utilized the default username and password combination such as `root` and `xc3511` respectively. These credentials are common on the Internet and can be used to access various types of DVRs.

I'm not sure about this. It makes it sound as though this is unexpected and an unknown vulnerability? Maybe something like:

> It was discovered that one of the reasons this was so easily done was that most IoT devices still use the default username and password combination that they were shipped with. Those credentials are easy to find on the Internet, and were probably used to compromise thousands of devices with minimal effort.

File: _posts/2016-10-25-how-the-biggest-attack-on-the-internet-was-perpertrated.markdown (hunk 1 above)
Comment (unicodeveloper) [inline _posts/2016-10-25-how-the-biggest-attack-on-the-internet-was-perpertrated.markdown]: That actually makes sense!

**Pull request merged**